src/crate/testing/layer.py

```diff
@@ -182,6 +184,8 @@ def __init__(self,
 
         self.process = None
         self.verbose = verbose
+        self.env = env
+        self.env.update(dict(CRATE_USE_IPV4='true'))
```

Comment:
Either use `self.env['CRATE_USE_IPV4'] = 'true'` because it's only setting a single value, or better `self.env.setdefault` to allow a user to explicitly override the default.